wiki/Medals/de.md

```diff
@@ -67,8 +62,9 @@ Obwohl die saisonalen Spotlights seit Frühjahr 2020 zugunsten des [aktuellen Be
 
 **Beatmap Challenge Packs Medaillen**, oder alternativ auch [*Skill Packs Medaillen*](https://osu.ppy.sh/home/news/2020-11-20-featured-artist-beatmap-updates-from-the-mappers-guild#skill-packs) genannt, sind Medaillen, die beim Abschließen der höchsten Schwierigkeitsstufe auf allen Beatmaps, die in einem bestimmten [Thema-Beatmap-Paket](https://osu.ppy.sh/beatmaps/packs?type=theme) oder [Beatmap-Paket eines Künstlers/Albums](https://osu.ppy.sh/beatmaps/packs?type=artist) enthalten sind, überreicht werden (*ohne aktivierte [Mods zur Reduzierung der Schwierigkeit](/wiki/Gameplay/Game_modifier#einfacher) oder [spezielle](/wiki/Gameplay/Game_modifier#sonstige) Mods*). Diese Beatmap-Pakete werden speziell angeboten, um das Verständnis von verschiedenen Gameplay-Aspekten um ein bestimmtes Fähigkeitslevel der Spieler zu testen, also z. B. innerhalb einer bestimmten [Sternebewertung](/wiki/Beatmap/Star_rating) oder einem Bereich von Schwierigkeitsgraden.
 
-Alle Beatmaps im Paket müssen im [osu!](/wiki/Game_mode/osu!)-Spielmodus abgeschlossen werden, um sich für das Abzeichen zu qualifizieren, sofern nicht anders angegeben.
+Für den Erwerb dieser Medaillen müssen alle Beatmaps im jeweiligen Paket im [osu!](/wiki/Game_mode/osu!)-Spielmodus abgeschlossen werden, sofern nicht anders angegeben.
 
 ## Siehe auch
 
+- [Ersterwerb von Medaillen](First_medal_unlocks)
```

Comment:
Not too sure on the translation here

Reply:
Hm.. might need to be a bit less literal for this one. `Erste Freischaltungen von Medaillen` just doesn't sound great. Maybe something like "Medaillen-Ruhmeshalle"? It is not quite as precise (could also be a list of who has the most medals) but sounds a bit more natural IMHO

Reply:
Let's go with `Ruhmeshalle`. People will need to check out the explanation in the referenced article anyway to find out more information.

```suggestion
- [Medaillen-Ruhmeshalle](First_medal_unlocks)
```